tldw_Server_API/app/core/Chat/command_router.py

```
@@ -0,0 +1,329 @@
+    except Exception as e:
+        return CommandResult(ok=False, command="time", content=f"Time lookup failed: {e}", metadata={"error": "time_error"})
+
+
+def _weather_handler(ctx: CommandContext, args: Optional[str]) -> CommandResult:
+    location = (args or "").strip()
+    if not location:
+        location = _cfg_str("DEFAULT_LOCATION", "default_location", "").strip()
+    # Importing the module ensures test monkeypatches to weather_providers.get_weather_client apply
+    client = weather_providers.get_weather_client()
+    try:
+        result = client.get_current(location=location or None)
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)